src/components/Common/Sidebar/ZekSidebar.vue

```diff
@@ -73,8 +74,6 @@
             :href="sec.title.url"
             :title="sec.title.tooltip || sec.title.label"
             class="link title"
-            @click.prevent="sec.title.isExpanded = !sec.title.isExpanded"
-            @click="$emit('onRoute', sec.title.url)"
```

Comment:
I don't see the removal of these lines necessary, especially the `$emit`, could you elaborate why were they removed? as they could be used in other projects

Reply:
its affect the href link, href does not work at this situation